packages/@lwc/integration-karma/test/rendering/sequential-reordering/index.spec.js

```diff
@@ -0,0 +1,45 @@
+import { createElement } from 'lwc';
+
+import Test from 'x/test';
+
+describe('vfragment sequential reordering', () => {
+    it('move fragment left (["foo", "fragment"] => ["fragment", "bar"])', async () => {
+        const elm = createElement('x-test', { is: Test });
+        document.body.appendChild(elm);
+        await Promise.resolve();
+
+        // Setup
+        elm.beforeItems = ['foo'];
+        elm.afterItems = [];
+        await Promise.resolve();
+
```

Comment:
```suggestion
        const initialContents = [...elm.shadowRoot.childNodes].map((_) => _.textContent);
        expect(initialContents).toEqual(['', 'foo', '', '', '一', '二', '三', '']);

```

Just for clarity (and out of a bit of paranoia), I think it would be great to assert what the state is before the mutation.